mixer2-localmail/src/main/java/org/mixer2/sample/Main.java

```diff
@@ -0,0 +1,75 @@
+package org.mixer2.sample;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.mixer2.Mixer2Engine;
+import org.mixer2.jaxb.xhtml.H1;
+import org.mixer2.jaxb.xhtml.Html;
+import org.mixer2.jaxb.xhtml.P;
+import org.mixer2.xhtml.exception.TagTypeUnmatchException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.apache.commons.mail.*;
+
+public class Main {
+	private static Logger logger = LoggerFactory.getLogger(Main.class);
+
+	// Server setting
+	private static final String HOST_NAME = "localhost";
+	private static final int PORT = 8080;
+
+	//
+	private static final String TO_ADDR = "nabedge@mixer2.org",
```

Comment:
"example" adddres is example.com . i.e, foobar@example.com . see http://www.iana.org/domains/reserved